openedx/core/djangoapps/user_api/views.py

```diff
@@ -232,37 +230,6 @@ def get(self, request):
         for field_name in self.DEFAULT_FIELDS:
             self.field_handlers[field_name](form_desc, required=True)
 
-        # Custom form fields can be added via the form set in settings.REGISTRATION_EXTENSION_FORM
-        custom_form = get_registration_extension_form()
-
-        if custom_form:
-            for field_name, field in custom_form.fields.items():
-                restrictions = {}
-                if getattr(field, 'max_length', None):
-                    restrictions['max_length'] = field.max_length
-                if getattr(field, 'min_length', None):
-                    restrictions['min_length'] = field.min_length
-                field_options = getattr(
-                    getattr(custom_form, 'Meta', None), 'serialization_options', {}
-                ).get(field_name, {})
-                field_type = field_options.get('field_type', FormDescription.FIELD_TYPE_MAP.get(field.__class__))
-                if not field_type:
-                    raise ImproperlyConfigured(
-                        "Field type '{}' not recognized for registration extension field '{}'.".format(
-                            field_type,
-                            field_name
-                        )
-                    )
-                form_desc.add_field(
-                    field_name, label=field.label,
-                    default=field_options.get('default'),
-                    field_type=field_options.get('field_type', FormDescription.FIELD_TYPE_MAP.get(field.__class__)),
-                    placeholder=field.initial, instructions=field.help_text, required=field.required,
-                    restrictions=restrictions,
-                    options=getattr(field, 'choices', None), error_messages=field.error_messages,
-                    include_default_option=field_options.get('include_default_option'),
-                )
-
```

Comment:
I didn't remove it myself, I think it happened when I git pulled from
master and was resolving merge conflicts.

On Sun, Apr 3, 2016 at 10:07 AM, Luiz Aoqui <notifications@github.com>
wrote:

> In openedx/core/djangoapps/user_api/views.py
> <https://github.com/bigdatauniversity/edx-platform/pull/3#discussion_r58306959>
> :
>
> > -                    raise ImproperlyConfigured(
> > -                        "Field type '{}' not recognized for registration extension field '{}'.".format(
> > -                            field_type,
> > -                            field_name
> > -                        )
> > -                    )
> > -                form_desc.add_field(
> > -                    field_name, label=field.label,
> > -                    default=field_options.get('default'),
> > -                    field_type=field_options.get('field_type', FormDescription.FIELD_TYPE_MAP.get(field.__class__)),
> > -                    placeholder=field.initial, instructions=field.help_text, required=field.required,
> > -                    restrictions=restrictions,
> > -                    options=getattr(field, 'choices', None), error_messages=field.error_messages,
> > -                    include_default_option=field_options.get('include_default_option'),
> > -                )
> > -
>
> @andela-tadesanya <https://github.com/andela-tadesanya> why was this
> section removed?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/bigdatauniversity/edx-platform/pull/3/files/b0f9c38a8a3aeb5cf61c7b328574a958bbd79673..002284bbf532ba067483af9a6283bccd15069574#r58306959>
>



-- 
*Oluwatosin Adesanya /** Andela.com
<http://andela.com/?utm_source=andela&utm_medium=email&utm_campaign=signature> *
*Software Developer / +234 813 845 1982 / @Andela
<http://bit.ly/andelatwitter>*

*"The startup that's harder to get into than Harvard" - *CNN
<http://snip.ly/HfId>
See what The Wall Street Journal <http://snip.ly/u9AI> has to say.